src/main/java/org/openmrs/maven/plugins/Run.java

```diff
@@ -0,0 +1,79 @@
+package org.openmrs.maven.plugins;
+
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.codehaus.plexus.components.interactivity.Prompter;
+import org.openmrs.maven.plugins.utility.AttributeHelper;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.InputStreamReader;
+
+/**
+ * @goal run
+ * @requiresProject false
+ */
+public class Run extends AbstractMojo {
+
+    private static final String NO_SERVER_TEXT = "There no server with given serverId. Please create it using omrs:setup first";
+
+    /**
+     * @parameter expression="${serverId}"
+     */
+    private String serverId;
+
+    /**
+     * @component
+     */
+    private Prompter prompter;
+
+    public void execute() throws MojoExecutionException, MojoFailureException {
+        AttributeHelper helper = new AttributeHelper(prompter);
+        ModuleInstall installer = new ModuleInstall(prompter);
+        File serverPath = installer.getServerPath(helper, serverId, NO_SERVER_TEXT);
+        try {
+            ProcessBuilder pb = new ProcessBuilder("mvn", "clean", "install");
```

Comment:
I decided to use this class because it is multi-platform issue. 
The other way is to use such:
Runtime.getRuntime().exec(myCommand);

but it requires to set the system shell (cmd for windows, bash for linux, etc), which is not good